apps/designer/package.json

```diff
@@ -15,7 +15,8 @@
     "storybook:run": "start-storybook -p 6006",
     "storybook:build": "build-storybook",
     "migrations": "migrations --dev",
-    "migrations:deploy": "migrations migrate"
+    "migrate": "migrations migrate --dev --force",
+    "migrations:deploy": "migrations migrate --force"
```

Comment:
these two don't read very intuitively, migrate vs migrations:deploy maps to the same migrate command but differentiates in. --dev, anything we can do about it? Or is it just me not seeing full picture?

Reply:
Yeah. It's kinda awkward. One is to allow to apply migrations from root for developers. Another is to be called during deployment by Vercel.

Would this be better?

```
"migrate": "migrations migrate  --force --dev",
"migrate:deploy": "migrations migrate --force"
```

Reply:
what's the difference when we do it at vercel?

Reply:
### `--dev`

This flag can be added to any command. It informs the CLI that it’s used in a development environment, which slightly changes its behaviour:

- automatically loads variables from `.env` files,
- uses a lockfile to avoid running more than one migration process at the same time.

Reply:
I was thinking about splitting `--dev` into two flags `--load-env` and `--use-lockfile`. But we'll still need these two separate scrips in `pakcage.json`

Reply:
Oh I read what --dev means, what I don't quite understand is how migrate:deploy is in the end different than "migrate"

Reply:
Maybe we don't actually need 2 separate scripts that a semantically slightly confusing if we for example:
- do dev mode by default and prod mode has to be passed via either NODE_ENV which we already have or --prod
- or separate flags that would turn it into prod mode


In the end we wouldn't need 2 scripts, it would be same script, just the CLI on vercel would have the right env/flags

Reply:
Not sure it's a good idea to rely on `NODE_ENV`. It's not always set up correctly.

Switching from `--dev` to `--prod` and calling with `--prod` at Vercel sounds good to me.

Reply:
cool, lets do this

Reply:
still don't quite understand how the prod flag will have a different outcome on the migrations :)

Reply:
After thinking a bit more, I think it's a bad idea to call with `--prod` at Vercel. Two reasons:

1. That locks us with `--prod`, and would not allow us to switch to `--load-env / --use-lockfile` later.
2. This makes Vercel know about our internal details of migrations.

I'd stick with two scripts in `package.json`. It's most correct in my opinion.